### ISSUE TITLE
[Issue] Selection to open in which container has more errors than reported

### ISSUE THREAD

**Issue by RealDavidoff** [mozilla/multi-account-containers#1388]
What I mean by "more errors than reported":
Both of these topics
https://github.com/mozilla/multi-account-containers/issues/1312
https://github.com/mozilla/multi-account-containers/issues/1222

Plus now mine, relate to the same issue IMHO. Because in addition to the other two I can also report a repetitive error for THIS situation:

- Twitter has assigned container, but is NOT set as default
- in another/different container I want to open twitter too of course
- am presented with: "Open this site in your assigned container?"
- now, sometimes the new tab can only be opened in the existing assigned container while the new container click does NOTHING
- and other times the new tab can only be opened in the new container, while clicking the assigned container does NOTHING.

I am sorry this is so ambigous/inconsistent, but it is (in my case at least).
I could not (yet) nail this plugin behavior down to specific websites, but I HAVE noticed this contradicting and in any case unintended/failing behavior many times.

Also a RE-openening of the new tab in the new container yields the same problem, is no remedy.

So basically, for me, it is currently NOT possible to open a new tab for the NEW container.
Which suggests: The error seems to be lying at the point of code that associates a website with an already assigned container?

I hope this noob advice helps, thank you. The feature is BRILLIANT in all other regards and I wouldn't want to miss it :-)

**Comment by RealDavidoff** [comment]
----
Brief update:
- 61 tabs are open, in case it matters (but almost all "auto tab discarded")
- and: I just found a WORKAROUND that *sometimes* works (I tried 5 times, twice it worked):

1) open new tab
2) click the "multi account containers" icon
3) now instead of clicking your new container, click the arrow to the right of it
4) then click that entry "multi account containers confirm" that shows weird code upon hovering
5) now enter the website, and (for me) it worked.

HOWEVER, trying to repeat this, it didn't work because there was no arrow anymore :-(
I closed the browser and reopened. Still the same.
I closed again and reopened. Now that arrow was back, I clicked it and proceeded as per 4) above.

So yes, sth is weird with the feature, but with a bit of persistence I can get it to work in a way that clearly isn't intended.
